lib/Raven/Configuration.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Raven;
+
+use Composer\CaBundle\CaBundle;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Configuration container for the Sentry client.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+class Configuration
```

Comment:
> And they have access.

They have access to the client instance, which in turn owns them. This is not necessarily bad (at least in not all cases), but it creates a circular reference. If we can avoid that, we should

>This is a logging library not a nuclear station's center.

Neither the other libraries, but they do it anyway because this way the code is better organized